### PR TITLE
backport (tempfile and weakref) new recipes

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: dev-python/backports-tempfile/backports_tempfile-1.0.recipe [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	${portName}_$pythonPackage = $portVersion\
+	\"; \
+REQUIRES_$pythonPackage=\"\
+	haiku\n\
+	cmd:python$pythonVersion\
+	\""
+BUILD_REQUIRES="$BUILD_REQUIRES
+	setuptools_scm_$pythonPackage"
+BUILD_PREREQUIRES="$BUILD_PREREQUIRES
+	cmd:git
```
Comment (korli) [inline dev-python/backports-tempfile/backports_tempfile-1.0.recipe:45]: cmd:git shouldn't be required

Comment (Begasus) [reply dev-python/backports-tempfile/backports_tempfile-1.0.recipe:45]: Build fails without it (seems a requirement for the source

Comment (korli) [reply dev-python/backports-tempfile/backports_tempfile-1.0.recipe:45]: did you try with SETUPTOOLS_SCM_PRETEND_VERSION?
https://github.com/haikuports/haikuports/blob/master/dev-python/iniconfig/iniconfig-1.1.1.recipe#L54

Comment (Begasus) [reply dev-python/backports-tempfile/backports_tempfile-1.0.recipe:45]: Not yet, will need to give it a run :) thanks for the pointer!

Comment (Begasus) [reply dev-python/backports-tempfile/backports_tempfile-1.0.recipe]: Works, thanks @korli !

**Pull request merged**